packages/js-example/README.md

```diff
@@ -5,14 +5,22 @@ Demonstrate how to import/decode an XML model using `Codecs`.
 ## Setup
 
 Initialize all packages
-> From the repository root, run `npm install`.
+From the repository root, run:
+```bash
+npm install
+cd packages/core/
+npm pack
```

Comment:
**thought:** runinng `npm run build` is enough.
There is no need to build the npm package tar.gz file.